.github/workflows/test-reporter.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     steps:
       - uses: elastic/apm-pipeline-library/.github/actions/test-report@current
         with:
-          artifact: test-results            # artifact name
-          name: JUnit Tests                 # Name of the check run which will be created
+          artifact: /test-results-(.*)/     # artifact name
+          name: 'Test Results $1'           # Name of the check run which will be created
```

Comment:
[For Reviewer]: The same approach was already used and tested in the `elastic-apm-agent`
https://github.com/elastic/apm-agent-java/blob/main/.github/workflows/test-reporter.yml